README.md

```diff
@@ -91,9 +87,25 @@ npm run test [component-name(s)]
 npm run pr
 ```
 
+### Documentation site
+Make sure you install the documentation site dependencies first.
+```shell
+cd docs && npm install
+```
+
+View the documentation locally
+```shell
+cd docs && npm start
+```
+
+Build the documentation site
+```shell
+npm run build-docs
```

Comment:
```suggestion
npm run docs:build
```

I'd love to standardize on using the colon approach for scripts.  It reads more intuitively in my opinion.

Reply:
I think we can leave it as-is for right now. All of the other scripts have a dash separating two words. We can revisit this later.